core/src/global.css

```diff
@@ -26,6 +26,8 @@ html:not(.x) {
 	--support-3: hsl(221, 100%, 61%);
 	--support-4: hsl(217, 85%, 49%);
 
+	--failure: hsla(0, 100%, 34%, 1);
+
```

Comment:
It's dangerous to define just one single color like this. In the original palette from Moai, there are 10 values of red: https://github.com/moaijs/moai/blob/57bef2433df8d1fa58b5b49e80e20492f7d0a709/core/src/global/global.css#L35-L44

What we should do here is to have this new value at the correct level i.e. not `--failure` but should be `--failure-X`. We find out what is X based on the lightness value in hsl format (so maybe you'll first need to convert the original ones in Moai to hsl and put them here)

Reply:
The second thing is that changing one value in a scale is usually not a good idea. We should change the whole scale. Look into the design, especially the design system and UI kit files, for more red values, and repeat the above step to try mapping them to failure-X. Then, we will manually fill the missing ones.